src/main/java/seedu/mamta/Parser.java

```diff
@@ -0,0 +1,90 @@
+package seedu.mamta;
+
+/**
+ * Parses user input and transforms it into a format suitable for further processing.
+ */
+public class Parser {
+
+    /**
+     * Transforms user output into a format suitable for further processing.
+     * @param userOutput The user's output.
+     * @return Transformed output for further processing.
+     */
+    public static String transformText(String userOutput) {
```

Comment:
Similar to above, could some code be abstracted to another function? The course handbook recommends functions be at most 30 lines long - while I feel 30 is a tight limit, perhaps some details could be hidden in other classes to improve readability for the user?